src/index.ts

```diff
@@ -352,6 +391,8 @@ function maybeDenull(o: GraphQLOutputType): GraphQLOutputType {
 export type Config = {
   scalarDefaults: Record<string, string>;
   taggedIds?: Record<string, string>;
+  typeDeclarationFilePath?: string;
```

Comment:
Hm, wdyt about just `typesFilePath` to shorten it up a little bit? But I'm fine with this if you think the explicitness is better.

Reply:
Updated!